common/fieldtypes.xml

```diff
@@ -6,13 +6,13 @@ Copyright (c) 2014, 2015 Wieland Hoffmann, Jeff Weeks
 <types>
 
   <fieldtype name="string" class="solr.StrField" sortMissingLast="false"/>
-  <fieldType name="long" class="solr.TrieLongField" precisionStep="0" positionIncrementGap="0"/>
+  <fieldType name="long" class="solr.LongPointField" positionIncrementGap="0"/>
   <fieldType name="mbid" class="solr.UUIDField" omitNorms="true" />
   <fieldType name="storefield" class="solr.StrField"/>
   <fieldType name="bool" class="solr.BoolField"/>
-  <fieldType name="date" class="solr.DateRangeField" sortMissingLast="false"/>
-  <fieldType name="int" class="solr.TrieIntField" sortMissingLast="false"/>
-  <fieldType name="float" class="solr.TrieFloatField"/>
+  <fieldType name="date" class="solr.DatePointField" sortMissingLast="false"/>

```

Comment:
From http://lucene.apache.org/solr/7_0_0/changes/Changes.html#v7.0.0.upgrading_from_solr_6.x I can see that the Trie fields are deprecated, but the same doesn't appear to be true for `DateRangeField`. In fact, https://lucene.apache.org/solr/guide/7_3/working-with-dates.html still talks about it being the date type that supports ranges.